docs/UserGuide.md

```diff
@@ -49,7 +52,7 @@ If you can type fast, Tutor's pet can get your class management tasks done faste
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/student` or as `n/John Doe`.
 
 * Items with `…`​ after them can be used multiple times including zero times.<br>
   e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/Experienced`, `t/Average t/Potential TA` etc.
```

Comment:
Need to update `tag` prefix here as well.

Reply:
Resolved by updating line 58 to 

`e.g. `[tag/TAG]…​` can be used as ` ` (i.e. 0 times), `tag/student`, `tag/TA` etc.`